weaviate/util.py

```diff
@@ -522,7 +523,8 @@ def check_batch_result(
     for result in results:
         if "result" in result and "errors" in result["result"]:
             if "error" in result["result"]["errors"]:
-                print(result["result"]["errors"])
+                logger = logging.getLogger("weaviate-client")
+                logger.debug("Batch result errors: %s", result["result"]["errors"])
```

Comment:
Very happy to see the prints here, and elsewhere, in the codebase replaced with logging, nice!